consensus/spos/bls/subroundEndRound_test.go

```diff
@@ -610,6 +629,152 @@ func TestSubroundEndRound_CheckSignaturesValidityShouldRetunNil(t *testing.T) {
 	assert.Equal(t, nil, err)
 }
 
+func TestSubroundEndRound_DoEndRoundJobByParticipant_RoundCanceledShouldReturnFalse(t *testing.T) {
```

Comment:
test name is to long maybe you can remove from test name SubroundEndRound because already the name of the file is tell you that you are testing SubroundEndRound

Reply:
No. I will leave it like this. Imagine the test fails in TeamCity. You won't be able to know where does this test belong. I will keep with the standard